target/linux/mpc85xx/image/p1020.mk

```diff
@@ -1,20 +1,35 @@
-define Device/aerohive_hiveap-330
+define Device/aerohive_hiveap
   DEVICE_VENDOR := Aerohive
-  DEVICE_MODEL := HiveAP-330
   DEVICE_PACKAGES := kmod-tpm-i2c-atmel
   BLOCKSIZE := 128k
   KERNEL := kernel-bin | gzip | uImage gzip
   KERNEL_SIZE := 8m
   KERNEL_INITRAMFS := copy-file $(KDIR)/vmlinux-initramfs | uImage none
   IMAGES := fdt.bin sysupgrade.bin
-  IMAGE/fdt.bin := append-dtb
+  IMAGE/fdt.bin := append-dtb | pad-to 256k | check-size 256k
+endef
+
+define Device/aerohive_hiveap-330
+  $(Device/aerohive_hiveap)
+  DEVICE_MODEL := HiveAP-330
   IMAGE/sysupgrade.bin := append-dtb | pad-to 256k | check-size 256k | \
 	append-uImage-fakehdr ramdisk | pad-to 256k | check-size 512k | \
 	append-rootfs | pad-rootfs $$(BLOCKSIZE) | pad-to 41216k | check-size 41216k | \
 	append-kernel | append-metadata
 endef
 TARGET_DEVICES += aerohive_hiveap-330
 
+define Device/aerohive_hiveap-370
+  $(Device/aerohive_hiveap)
+  DEVICE_MODEL := HiveAP-370
+  DEVICE_PACKAGES += kmod-ath10k-ct ath10k-firmware-qca988x-ct kmod-ath9k nand-utils
```

Comment:
This also needs the `block-mount` package in order to mount the `rootfs_data` volume.

Reply:
`block-mount` has now been added; `rootfs_data` now automounts.